### PR TITLE
Update Kevin Instructor landing page

### PR DESCRIPTION
**Pull request by zacjones93**
- sourced from sanity

[Preview](https://egghead-io-nex-git-zj-kevin-cunningham-landing-page-egg-72a498.vercel.app/q/resources-by-kevin-cunningham)

painting

Learn web development from Kevin Cunningham on egghead  egghead i
Learn web development from Kevin Cunningham on egghead  egghead i

### CODE REVIEW

File: src/components/search/instructors/kevin-cunningham/index.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    	path,
+      byline,
+    	image,
+      'background': images[label == 'feature-card-background'][0].url,
+      'instructor': collaborators[]->[role == 'instructor'][0]{
+      	'name': person->.name
+    	},
+    }
+  },
+}`
```
Comment (zacjones93) [inline src/components/search/instructors/kevin-cunningham/index.tsx:58]: sanity query

File: src/components/search/instructors/index.tsx [hunk 2]
```diff
@@ -20,6 +21,7 @@ const InstructorsIndex: any = {
   'stephanie-eckles': SearchStephanieEckles,
   'kamran-ahmed': SearchKamranAhmed,
   'alex-reardon': SearchAlexReardon,
+  'kevin-cunningham': SearchKevinCunningham,
```
Comment (zacjones93) [inline src/components/search/instructors/index.tsx:24]: add custom instructor landing page component to the index to _actually_ render

File: src/components/search/instructors/kevin-cunningham/index.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            <div className="flex flex-col items-center justify-center sm:space-x-5 sm:space-y-0 space-y-5 gap-10 mt-10">
+              <div className="flex-shrink-0">
+                <Link href={path}>
+                  <a
+                    tabIndex={-1}
+                    onClick={() =>
+                      track('clicked kevin cunningham page course CTA', {
+                        resource: path,
+                        linkType: 'image',
+                      })
```
Comment (zacjones93) [inline src/components/search/instructors/kevin-cunningham/index.tsx:97]: analytics

File: src/lib/instructors.ts [hunk 4]
```diff
@@ -52,6 +53,7 @@ const sanityInstructorHash = {
   'stephanie-eckles': stephanieEcklesQuery,
   'kamran-ahmed': kamranAhmedQuery,
   'alex-reardon': alexReardonQuery,
+  'kevin-cunningham': kevinCunninghamQuery,
```
Comment (zacjones93) [inline src/lib/instructors.ts:56]: add sanity query to the query index to fetch instructor data on landing page

File: src/components/search/instructors/kevin-cunningham/index.tsx [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+      >
+        <div className="relative z-10 px-5 sm:py-16 py-10 sm:text-left text-center">
+          <div className="space-y-5 mx-auto flex items-center justify-center max-w-screen-xl">
+            <div className="flex flex-col items-center justify-center sm:space-x-5 sm:space-y-0 space-y-5 gap-10 mt-10">
+              <div className="flex-shrink-0">
+                <Link href={path}>
+                  <a
+                    tabIndex={-1}
+                    onClick={() =>
+                      track('clicked kevin cunningham page course CTA', {
```
Comment (joelhooks) [inline src/components/search/instructors/kevin-cunningham/index.tsx:94]: nitpick: `clicked instuctor page CTA` where `instructor` is property is another way to look at this and also some meta about what the cta is like `type: 'featured course'` for similar.

**Review by joelhooks — APPROVED** [review]

File: src/components/search/instructors/kevin-cunningham/index.tsx (hunk 5 above)
Comment (zacjones93) [reply src/components/search/instructors/kevin-cunningham/index.tsx:94]: That makes sense, thanks.

**Pull request merged**